packages/public/server/src/module/Ui/templates/entity/unrevised.twig

```diff
@@ -21,6 +21,13 @@
  #}
 <div class="editor-design">
 {{ pageHeader('Unrevised learning resources').render() }}
+<h2>{% trans %} Support links for reviewing {% endtrans %}</h2>
+<ul>
+  {% for helpLink in helpLinks %}
+    {% set url, title = helpLink[0], helpLink[1] %}
+    <li><a href="{{ url }}">{{ title | trans }}</a></li>
```

Comment:
That won't work since title is dynamic

Reply:
Fixed with https://github.com/serlo/serlo.org/pull/164/commits/f38421482a664e3b94ce052ddc348c61817a0ee4